tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -608,6 +608,20 @@ describe(__filename, () => {
         slug: 'anime',
         type: ADDON_TYPE_STATIC_THEME,
       },
+      {
+        ...fakeCategory,
+        application: CLIENT_APP_ANDROID,
+        name: 'Alerts & Update',
+        slug: 'alert-update',
+        type: ADDON_TYPE_DICT,
+      },
+      {
+        ...fakeCategory,
+        application: CLIENT_APP_ANDROID,
+        name: 'Alerts & Update',
+        slug: 'alert-update',
+        type: ADDON_TYPE_LANG,
+      },
```

Comment:
Can we have a test/constant with `CLIENT_APP_FIREFOX` too?

Reply:
I suppose we could but `clientApp` does not play any role for the bug fixed here so we'll end up with duplicate tests essentially.